### PR TITLE
Remove table colors and use palette instead

### PR DESCRIPTION
**Pull request by wfortin**
Applied avocode specs, and removed duplicate or old colors to use the basic color palette.

### CODE REVIEW

File: scss/tables/table-actions.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @import '../common/colors';
 @import '../controls/filter-box';
 $coveo-table-actions-container-height: 52px;
-$coveo-table-actions-border-width: 2px;
+$coveo-table-actions-border-width: 1px;
```
Comment (marilynechan) [inline scss/tables/table-actions.scss]: Why did you need to change this width in more thant one place? I'm referring to the border-width you changed in fixed-header-table.

File: scss/tables/table.scss [hunk 2]
```diff
@@ -43,14 +43,8 @@ table {
...
 
-      border-top: solid 2px $table-borders;
-      border-bottom: solid 2px $table-borders;
-
-      &.metric {
-        color: $heading-2-color;
-
-        background-color: $metric-table-header-color;
-      }
+      border-top: solid 1px $medium-grey;
```
Comment (marilynechan) [inline scss/tables/table.scss]: Again changing from 2px to 1px.

**Comment by marilynechan** [comment]
Nice cleanup! Ship it!

File: scss/tables/table-actions.scss (hunk 1 above)
Comment (wfortin) [inline scss/tables/table-actions.scss]: Good point i'll make a variable $table-border in table.scss

**Comment by JPLachance** [comment]
Look's good. Ship it!
